pagopa/io-app
# chore: [IOCOM-1564] FIMS folder factor refactoring to enable sub-features

## Short description
refactored FIMS folder to follow the same organization structure as idPay
## List of changes proposed in this pull request
- moved current FIMS folder into the `SSO` sub-folder
- added `common`and `history` folders for organization's sake
- moved common logic in its own folder
- refactored imports accordingly

## How to test
running the dev-server, run the FIMS flow and make sure it all works as intended
